messages/template.list.json

```diff
@@ -0,0 +1,9 @@
+{
+  "description": "retrieves the list of templates available in your org\nSee 'Which Experience Cloud Template Should I Use?' in Salesforce Help for more information about the different template types available for Experience Cloud.",
```

Comment:
While the --help command-line output for this command displays the sentence that starts "See 'Which Experience Cloud....", the generated CLI Reference output file doesn't.  I don't know why it's failing -- the "description" property looks correct.  

Can you run it and see what you get? 

- Install the command reference generator plugin:
     `$ sfdx plugins:install @salesforce/plugin-command-reference`
- Generate XML files for the community commands:
    `sfdx commandreference:generate --plugins community -d output`
- Check the file output/force/community/cli_reference_force_community_template_list.xml : It should contain a Help section with the sentence "See 'Which Experience...."